src/Config_Command.php

```diff
@@ -349,6 +348,253 @@ private static function get_wp_config_vars() {
 		return array_merge( $wp_config_vars, $wp_config_constants, $wp_config_includes_array );
 	}
 
+	/**
+	 * Sets the value of a specific constant or variable defined in wp-config.php file.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <name>
+	 * : Name for the wp-config.php constant or variable.
+	 *
+	 * <value>
+	 * : Value to set the wp-config.php constant or variable to.
+	 *
+	 * [--add]
+	 * : Add the value if it doesn't exist yet.
+	 * This is the default behavior, override with --no-add.
+	 *
+	 * [--raw]
+	 * : Place the value into the wp-config.php file as is, instead of as a quoted string.
+	 *
+	 * [--anchor=<anchor>]
+	 * : Anchor string where additions of new values are anchored around.
+	 * Defaults to "/** Absolute path to the WordPress directory".
+	 *
+	 * [--placement=<placement>]
+	 * : Where to place the new values in relation to the anchor string.
+	 * ---
+	 * default: 'before'
+	 * options:
+	 *   - before
+	 *   - after
+	 * ---
+	 *
+	 * [--separator=<separator>]
+	 * : Separator string to put between an added value and its anchor string.
+	 * The following escape sequences will be recognized and properly interpreted: '\n' => newline, '\r' => carriage return, '\t' => tab.
+	 * Defaults to a single EOL ("\n" on *nix and "\r\n" on Windows).
+	 *
+	 * [--type=<type>]
+	 * : Type of the config value to set. Defaults to 'all'.
+	 * ---
+	 * default: all
+	 * options:
+	 *   - constant
+	 *   - variable
+	 *   - all
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Set the WP_DEBUG constant to true.
+	 *     $ wp config set WP_DEBUG true --raw
+	 *
+	 * @when before_wp_load
+	 */
+	public function set( $args, $assoc_args ) {
+		$path = $this->get_config_path();
+
+		list( $name, $value ) = $args;
+		$type = Utils\get_flag_value( $assoc_args, 'type' );
+
+		$options = array();
+
+		$option_flags = array(
```

Comment:
Okay I see you changed the library defaults but this is non-transparent and frankly I don't like this whole loop block technique but will let it go....!